src/components/config-sessions.tsx

```
@@ -0,0 +1,407 @@
+}
+
+function buildPreviewCapped(ids: string[], items: SessionFileInfo[], t: (k: string, o?: unknown) => string) {
+  const raw = ids.slice(0, 3).map(id => truncatePreview(items.find(s => s.id === id)?.title || id))
+  const preview = raw.join('、')
+  const more = ids.length > 3 ? t('sessions.delete.batch_more', { count: ids.length - 3 }) as string : ''
+  const text = preview + more
+  return text.length > 72 ? `${text.slice(0, 72)}…` : text
+}
+
+
+export function ConfigSessions() {
```

ESLint

[error] More than 1 blank line not allowed.

(style/no-multiple-empty-lines)